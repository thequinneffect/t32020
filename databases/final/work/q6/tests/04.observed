tests/04.sh: 1: ./q6: Permission denied
