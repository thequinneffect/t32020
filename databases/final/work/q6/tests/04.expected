Discography for Roaring Buffalo
--------------------
Rocking Blue (2005) (hip-hop)
--------------------
 1. Rocking this City (12:14)
 2. On the Road to the Dance Floor (4:09)
 3. Take the Ocean (10:43)
 4. Bring the Torpedoes (8:58)
 5. Master of the Sun (6:05)
 6. Get out of Tokyo (5:00)
--------------------
Shock Midnight (2006) (rock)
--------------------
 1. Rocking the Moon (5:22)
 2. Take Life (6:29)
 3. Fly to the Dance Floor (8:07)
 4. View to this Night (8:00)
 5. On the Road to the Universe (10:22)
 6. Pump up Haze (5:42)
 7. Bad for Vision (5:30)
 8. Where is this City (5:05)
 9. Jump on the Clouds (3:23)
10. Waiting for this City (5:40)
--------------------
On the Road to Vision (2007) (rock)
--------------------
 1. Roll with Life (4:58)
 2. Force this World (5:01)
 3. Where is the Moon (6:39)
 4. Why not Plague (4:44)
 5. Over the Horizon (4:49)
--------------------
What about Destruction (2008) (rock)
--------------------
 1. What happened to the Sun (5:47)
 2. Up to the Heat (11:27)
 3. Kill for Destruction (5:12)
 4. Get out of the Nights (4:46)
 5. Running from the Clouds (3:41)
 6. What is Love (4:43)
 7. What happened to the Sky (6:40)
--------------------
Feel Vision (2011) (country)
--------------------
 1. Fly to Pandemic (4:34)
 2. Maybe the Bed (11:30)
 3. Rock Fear (4:32)
 4. Force Triumph (8:23)
 5. View to the Bed (10:58)
 6. Fly to the Nights (6:27)
 7. Rock the Heat (1:59)
 8. Rock Blue (6:53)
--------------------
Bad for Triumph (2014) (pop)
--------------------
 1. On the Road to Solace (2:40)
 2. Get out of the Horizon (3:55)
 3. Jump on the Horizon (5:48)
 4. Running from the Universe (5:04)
 5. Fast Train to Blue (3:26)
 6. What is the Sun (3:35)
 7. Force Fear (6:34)
 8. Dreams of Life (5:33)
 9. Touch the Heat (4:28)
--------------------
Where is this World (2016) (pop)
--------------------
 1. What about Love (5:08)
 2. Rock the Heat (4:45)
 3. Where is Defeat (3:16)
 4. Jump on the Universe (5:04)
 5. Ode to the Clouds (3:42)
 6. Fast Train to the Torpedoes (3:24)
 7. Ode to Pandemic (4:41)
 8. What is the Bed (5:11)
 9. Where is Midnight (5:56)
10. On the Road to this World (6:38)
--------------------
Bad for the Sea (2019) (rock)
--------------------
 1. Dreams of the Farm (1:44)
 2. What about the Sea (5:44)
 3. Rocking Khatmandu (4:34)
 4. Waiting for Fire (7:56)
 5. Feel the Bed (3:51)
 6. Where is the Ocean (5:47)
--------------------
Over the Nights (2020) (rock)
--------------------
 1. Ode to the Farm (8:17)
 3. Where is Destruction (9:57)
 4. Where is Fear (4:34)
 5. Where is the Universe (5:46)
